tools/unsupervised_dataset/Crawler/run_crawler.py

```diff
@@ -0,0 +1,16 @@
+import requests
+import sys
+import Crawler
+
+# Initializes a crawler and starts the crawling process using command line arguments
+def start_crawler():
+    # TO-DO: Allow user to optionally define max size or max depth
```

Comment:
nit: seems style guide recommends us using: # TODO(kl@gmail.com): Use a "*" here for string repetition.